From: Shengjing Zhu <zhsj@debian.org>
Date: Tue, 14 Jul 2020 01:08:24 +0800
Subject: disable test failed with reflectwalk v1.0.1

---
 template/interpolate/render_test.go | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/template/interpolate/render_test.go b/template/interpolate/render_test.go
index ae9ff24..cab17e8 100644
--- a/template/interpolate/render_test.go
+++ b/template/interpolate/render_test.go
@@ -86,24 +86,6 @@ func TestRenderMap(t *testing.T) {
 			nil,
 		},
 
-		// this test fails if you get github.com/mitchellh/reflectwalk@v1.0.1
-		// the fail is caused by
-		// https://github.com/mitchellh/reflectwalk/pull/22/commits/51d4c99fad9e9aa269e874bc3ad60313a574799f
-		// TODO: open a PR to fix it.
-		"nested value keys": {
-			map[string]interface{}{
-				"foo": map[string]string{
-					"{{upper `bar`}}": "{{upper `baz`}}",
-				},
-			},
-			map[string]interface{}{
-				"foo": map[string]string{
-					"BAR": "BAZ",
-				},
-			},
-			nil,
-		},
-
 		"filter": {
 			map[string]interface{}{
 				"bar": "{{upper `baz`}}",
